File: java_problems/src/UnderThree.java
import java.util.Scanner;

public class UnderThree {
    public static void main(String[] args) {
        // instances
        Scanner sc = new Scanner(System.in);

        // vars
        int a, b, c, lower_value;

        // input
        System.out.println("Integer values:");
        System.out.print("First value: ");
        a = sc.nextInt();
        System.out.print("Second value: ");
        b = sc.nextInt();
        System.out.print("Third value: ");
        c = sc.nextInt();
        System.out.println();

        // logic
        if (a < b && a < c) {
            lower_value = a;
        }
        else if (b < c) {
            lower_value = b;
        }
        else {
            lower_value = c;
        }

        // results
        System.out.print("Lower value -> " + lower_value + ".");

        sc.close();
    }
}

File: cpp_problems/UnderThree.cpp
#include <iostream>

using namespace std;

int main()
{
    // vars
    int a, b, c, lower_number;

    // inputs
    cout << "Enter three integer:" << endl;
    cout << "First value: ";
    cin >> a;
    cout << "Second value: ";
    cin >> b;
    cout << "Third value: ";
    cin >> c;
    cout << endl;

    // logic
    if (a < b && a < c) {
        lower_number = a;
    }
    else if (b < c) {
        lower_number = b;
    }
    else {
        lower_number = c;
    }

    // answers
    cout << "The smallest value reported is " << lower_number << ".";

    return 0;
}